charts/kubechecks-rbac/values.yaml

```diff
@@ -1,13 +1,6 @@
-roleName: "kubechecks-remote-role"
-roleBindingName: "kubechecks-remote-role-binding"
-namespace: "argocd"
-
-# for AWS EKS clusters
-eks:
-  # The ARN of the IAM role to bind to the Role
-  iamRoleArn: ""
+ClusterRoleName: "kubechecks-remote-role"
+ClusterRoleBindingName: "kubechecks-remote-role-binding"
+ClusterRoleBindingGroup: "kubechecks-remote-group"
```

Comment:
these should start with a lowercase letter, for consistency

Reply:
good point, changed to lowercase